core/templates/I18nFooter.ts

```diff
@@ -50,7 +52,16 @@ angular.module('oppia').directive('i18nFooter', [
           };
           ctrl.$onInit = function() {
             ctrl.supportedSiteLanguages = SUPPORTED_SITE_LANGUAGES;
-
+            // Fetching the saved language code from cookie as $translate in
+            //  angularjs uses cookies to store the last used code and we need
+            // fetch that code in order to have constistency in translations in
+            // both angular and angularjs.
+            if (document.cookie.includes('NG_TRANSLATE_LANG_KEY=')) {
```

Comment:
should it be $document?

Should NG_TRANSLATE_LANG_KEY= be a named constant if it's used elsewhere in the codebase?

Reply:
While this code is inside an angularjs directive, it only used to update angular values. So using $document wouldn't be necessary. `NG_TRANSLATE_LANG_KEY=` is not used anywhere else and will not be used anywhere else.

Reply:
We can, but then we will have to revert back to this when we migrate this file to angular, so I didn't use $document or $cookies for this.